test/sample_consensus/test_sample_consensus_line_models.cpp

```diff
@@ -149,6 +149,88 @@ TEST (SampleConsensusModelLine, OnGroundPlane)
   EXPECT_NEAR (0, coeff[5], 1e-4);
 }
 
+TEST (SampleConsensusModelLine, SampleValidationPointsEqual)
+{
+  PointCloud<PointXYZ> cloud;
+  cloud.resize (2);
+
+  cloud[0].getVector3fMap () <<  0.1f,  0.0f,  0.0f;
+  cloud[1].getVector3fMap () <<  0.1f,  0.0f,  0.0f;
+
+  const auto previous_verbosity_level = pcl::console::getVerbosityLevel();
+  pcl::console::setVerbosityLevel(pcl::console::L_VERBOSE);
```

Comment:
This should usually not be necessary, or do you have a specific reason to add this?
We did this at on place because a test sometimes failed randomly on the CI, but always ran successfully locally. If there is no reason to assume that this test might fail, it would just add unnecessary output

Reply:
The main reason I did this was to get/give an idea of the changed log output. The checks are now only implemented in `isSampleGood` which is then called from `computeModelCoefficients`. That implementation detail is leaked by the log messages. I don't know if that's a real issue or not.